pkgs/os-specific/linux/rtl88x2bu/default.nix

```diff
@@ -25,6 +25,9 @@ stdenv.mkDerivation rec {
 
   preInstall = ''
     mkdir -p "$out/lib/modules/${kernel.modDirVersion}/kernel/net/wireless/"
+
+    # Install 88x2bu.conf so it works in Linux 6.2 or newer.
+    install -D -m 644 $src/88x2bu.conf $out/etc/modprobe.d/88x2bu.conf
```

Comment:
I meant that is this file automatically added into `/etc/modprobe.d` (without adding it to `boot.extraModprobeConfig`.

Reply:
I want to thank you for raising this. But, indeed, it doesn't work. I tried to find a way to blacklist it, but it doesn't seem to exist, except by using a NixOS module.

I've removed this install command, and the user must block the module by hand.